test/determinism.js

```diff
@@ -1,16 +1,23 @@
 const tape = require('tape');
 const fs = require('fs');
 const solc = require('../index.js');
+const semver = require('semver');
 
 tape('Deterministic Compilation', function (t) {
   t.test('DAO', function (st) {
     var input = {};
     var prevBytecode = null;
+    var testdir;
+    if (semver.lt(solc.semver(), '0.5.0')) {
+      testdir = 'test/DAO040/';
+    } else {
+      testdir = 'test/DAO/';
```

Comment:
Worth mentioning that due to the current `develop` branch of solidity not using `0.5.0`, this change on its own will not be enough to make it pass. However we can just delete the `DAO040` directory an symlink `DAO` to it in our test suite.

Reply:
Yes, I was considering using 0.4.25 instead, but using 0.5.0 probably makes more sense, but we have to deal with this somehow.